src/Lime.Transport.Tcp/LimeTcpActivitySource.cs

```diff
@@ -0,0 +1,20 @@
+using System.Diagnostics;
+
+namespace Lime.Transport.Tcp
+{
+    /// <summary>
+    /// Provides the ActivitySource for the Lime.Protocol TCP transport
+    /// </summary>
+    public static class LimeTcpActivitySource
+    {
+        /// <summary>
+        /// The name of the ActivitySource for the Lime.Protocol TCP transport
+        /// </summary>
+        public const string LimeTcp = "Lime.Protocol.Tcp";
```

Comment:
I think we could match this string with the project name (which is also the Nuget package name): `Lime.Transport.Tcp`